### ISSUE TITLE
Add allowed ranges to datepicker

### ISSUE THREAD

**Issue by marosseleng** [marosseleng/compose-material3-datetime-pickers#18]

**Comment by kevinldavis** [comment]
I think this change caused the following error (jquery.ui 1.13.2) datepicker:
![Screenshot 2022-10-19 102955](https://user-images.githubusercontent.com/107428870/196736541-501cb030-3f7b-497a-a0b2-6808ebc1008d.jpg)

code:
         $(function () {
              $("#txtDateOfBirth").datepicker({
                  yearRange: { 1901: 2023 },
                  dateFormat: 'mm/dd/yyyy',
                  changeMonth: true,
                  changeYear: true
              });
          });

**Comment by marosseleng** [comment]
Hi @kevinldavis 
That problem is not related to this issue/repository as this is not a javascript library.

**Comment by kevinldavis** [comment]
Thank you

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by danieldaeschle** [comment]
I could need this feature. Are there any plans to implement it?

**Comment by marosseleng** [comment]
@danieldaeschle honestly, there aren't any for the near future.

Google created date & time pickers as part of `compose-material3`, it is currently in beta. 

I, personally, haven't seen, nor tried it, thus I am not sure which (planned) features of this library are actually reasonable to implement, when now there is "official" library.

I'm sorry, I can't give you a clear answer on this, but it would need a bit of research on this topic from my side.

**Comment by danieldaeschle** [comment]
Ah, didn't know that there is a datepicker yet. Must be new :)

**Comment by marosseleng** [comment]
Yeah, it was released in alpha few weeks ago, is currently in beta (https://developer.android.com/jetpack/androidx/releases/compose-material3#1.1.0-beta02)

**Comment by drmughal03244** [comment]
@marosseleng @danieldaeschle  will u please tell me how to disable Previous dates From Now in This Date Picker

**Comment by marosseleng** [comment]
@drmughal03244 As you can see, this issue is still open, thus it is currently not possible to disable previous dates in the datepicker.
